fungal-acne.html

```diff
@@ -75,15 +75,15 @@ <h1>Similar articles</h1>
                 <!-- Flexbox to divide the three recommended articles easier-->
                 <div class="flex-container">
                     <div class="flex-child"><img src="images/acne.png"></div>
-                    <div class="flex-child"><img src="images/clear-oily-skin.jpg"></div>
-                    <div class="flex-child"><img src="sample.png"></div>
+                    <div class="flex-child"><img src="images/oily/clear-oily-skin.jpg"></div>
+                    <div class="flex-child"><img src="images/dry/help-dry-skin.jpg"></div>

```

Comment:
I added another article in the flex box so now the "similar articles" section is complete for this page also.